src/config/dynamic-theme-fixes.config

```diff
@@ -16165,6 +16165,28 @@ CSS
 
 ================================
 
+q.utoronto.ca
+
+INVERT
+.ic-sidebar-logo__image
+.instructure_file_link_holder > .file_download_btn
+a.external > span > img:first-child
+
+CSS
+:root {
+  --ic-brand-global-nav-ic-icon-svg-fill--active: var(--darkreader-text--ic-link-color) !important;
+  --darkreader-bg--ic-brand-global-nav-bgd: var(--ic-brand-global-nav-bgd) !important;
+  --darkreader-bg--ic-brand-global-nav-logo-bgd: var(--ic-brand-global-nav-logo-bgd) !important;
+}
+.ic-app-header__menu-list-link:hover, .ic-app-header__menu-list-link:focus {
+  background-color: ${rgba(0, 0, 0, 0.2)} !important;
+}
+.ic-DashboardCard {
+  border: 1px ${#bbb} solid !important;
```

Comment:
```suggestion
  border-color: ${#bbb} !important;
```

Reply:
The original CSS has no border declared (it only uses a shadow) so all the properties need to be set explicitly.

Reply:
Oh in that case we should fix the shadow instead of adding a bordr.

Reply:
Okay, I've changed it to use the same box shadow as the original site, but with a 1px vertical shift (it looked a bit weird in dark mode with the original 2px) and the original color set up to invert.